proxy/proxy-standalone/main.go

```diff
@@ -0,0 +1,154 @@
+package main
+
+import (
+	"crypto/tls"
+	"encoding/json"
+	"encoding/pem"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	_ "net/http/pprof"
+	"os"
+	"os/exec"
+	"os/signal"
+	"runtime"
+	"runtime/pprof"
+
+	"github.com/nccgroup/tracy/api/rest"
+	"github.com/nccgroup/tracy/api/store"
+	"github.com/nccgroup/tracy/configure"
+	"github.com/nccgroup/tracy/log"
+	"github.com/nccgroup/tracy/proxy"
+)
+
+var cpuprofile = flag.String("cpuprofile", "", "write cpu profile to file")
+
+func main() {
+	if *cpuprofile != "" {
+		defer pprof.StopCPUProfile()
+	}
+
+	go func() {
+		log.Error.Fatal(rest.Server.ListenAndServe())
+	}()
+
+	fmt.Printf("Tracer server:\t%s%s",
+		configure.Current.TracyServer.Addr(), log.NewLine)
+
+	if configure.Current.AutoLaunch {
+		processAutoLaunch()
+	}
+
+	// Wait for the user to close the program.
+	signalChan := make(chan os.Signal, 1)
+	cleanupDone := make(chan bool)
+	signal.Notify(signalChan, os.Interrupt)
+	go func() {
+		for range signalChan {
+			fmt.Println("Ctrl+C pressed. Shutting down...")
+			store.DB.Close()
+			cleanupDone <- true
+		}
+	}()
+	<-cleanupDone
+}
+
+func init() {
+	// Parse the flags. Have to parse them hear since other package
+	// initialize command line.
+	flag.Parse()
+
+	// Set up the logging based on the user command line flags.
+	log.Configure()
+
+	// Set up the configuration.
+	configure.Setup()
+
+	if *cpuprofile != "" {
+		f, err := os.Create(*cpuprofile)
+		if err != nil {
+			log.Error.Fatal(err)
+		}
+		pprof.StartCPUProfile(f)
+	}
+
+	// // Open the database.
+	// if err := store.Open(configure.Current.DatabasePath, log.Verbose); err != nil {
+	// 	log.Error.Fatal(err.Error())
+	// }
+
+	// Initialize the HTTP routes.
+
+	oldHost := configure.Current.TracyServer.Hostname
+	oldPort := configure.Current.TracyServer.Port
+
+	configure.Current.TracyServer.Hostname = "127.0.0.1" //This is a hack to fix the fact we don't have an option to config just the proxy. Need to talk to Jake about how to fix this for real
```

Comment:
We can add configuration pretty easy. We can just have something like an optional proxy-only settings and if its set, then split the binaries or something. This should be fine for now, but can you open a issue for the new configuration option.